frontend/src/locales/README.md

```diff
@@ -0,0 +1,25 @@
+# Localizations
```

Comment:
If we really need this, please move this into `Code of Conduct` https://github.com/gradido/gradido/community

![image](https://user-images.githubusercontent.com/1238238/203608166-e7b9b493-6201-4bfc-8823-d97110878a13.png)

Reply:
Moved into `CONTRIBUTING.md`